fonduer/parser/simple_tokenizer.py

```diff
@@ -9,7 +9,7 @@
 class SimpleTokenizer(object):
     """Tokenizes text on whitespace only using split()."""
 
-    def __init__(self, delim):
+    def __init__(self, delim="<NB>"):
```

Comment:
This allows us to eliminate the duplicated `self.delim = "<NB>"` when we refactor the parser later.